HACKING.rst

```diff
@@ -28,6 +31,10 @@ you may need to run::
 
   pip install importlib
 
+You may get an error 'psycopg2 package not found' when you do 'haas init_db' 
+in the next step if you are using PostgreSQL database. You may need to run::
+
+  pip install psycopg2
```

Comment:
Rather than having an explicit pip command, perhaps we should just place the requirement in setup.py? It wouldn't hurt anything to have the extra package installed for sqlite users.